tests/smoke.sh

```diff
@@ -116,18 +141,22 @@ info "Starting node one by one and verifying if all services for them are runnin
 start_auxiliary_node 1406
 grep_try 1406 geth
 rpc_node_try 1406
-rpc_auxiliary_sub_graph_try 1406
+deploy_subgraph ropsten 1406 auxiliary 9426 6407
```

Comment:
We can add the hardcoded values to a variable for readability.
Right now it's hard to know what these hardcoded values are.